requirements.txt

```diff
@@ -1,2 +1,2 @@
-redpipe>=2.3.4
+redpipe=2.3.5
```

Comment:
shouldn't this be `>=` or `==` ?

Reply:
my preference would be `>=` since it is a library that is used by other projects and other projects may want to use a later version of redpipe at some point.

Reply:
oh yes sorry! thanks for catching it -- I meant to do that and ended up typing the wrong thing :)